src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else if (a.toString() < b.toString()) {
+        return -1;
+      } else {
+        return 0;
+      }
+    },
+  ) {
+    let tempValue;
+
+    for (let i = 0; i < this.length; i++) {
+      for (let k = 0; k < this.length - 1; k++) {
+        if (compareFunction(this[k], this[k + 1]) > 0) {
```

Comment:
Using `Array.prototype.sort` usually requires handling the case where the compareFunction is not provided in a way that it sorts the elements as strings by default. Your implementation does this, but it's worth noting that converting every element to a string can be inefficient for large arrays with non-string types.